### ISSUE TITLE
Umlauts not working

### ISSUE THREAD

**Issue by maxmarkus** [shprink/trendingdevs.com#14]
The parser cannot handle cities like  München  (Munich), so if someone has München in his githup account, he will not be listed.

**Comment by shprink** [comment]
Yes we query only with the English name so far but we plan to query with
the local name. Stay tuned
